### PR TITLE
Remove stock images from Project Index

### PR DESCRIPTION
**Pull request by jdalphon**
As per yesterdays meeting, we are finally removing the stock images from the projects page. 

1) Removed images from the repo
2) Added css for vertical centering of title where images are missing
3) Added titles where images were missing
4) Updated a test that was looking for a link that is no longer there.

### CODE REVIEW

File: app/assets/stylesheets/projects.css.scss [hunk 1]
```diff
@@ -46,10 +42,18 @@
...
+    margin-bottom: 12px;
   }
   
   .wide-input {
     width: 100% !important;
   }
+  
+  .placeholder{
+    font-size: 2em;
+    color:gray;
```
Comment (kcarcia) [inline app/assets/stylesheets/projects.css.scss]: space between color: and gray

File: app/assets/stylesheets/projects.css.scss [hunk 2]
```diff
@@ -46,10 +42,18 @@
...
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+  }
 }
```
Comment (kcarcia) [inline app/assets/stylesheets/projects.css.scss]: CSS rules are not alphabetized in this file

File: app/views/shared/_project_box.html.erb [hunk 3]
```diff
@@ -1,25 +1,38 @@
+<% has_img = @box_project.featured_media_id != nil%>
```
Comment (kcarcia) [inline app/views/shared/_project_box.html.erb]: space between nil and %>

File: app/assets/stylesheets/projects.css.scss (hunk 2 above)
Comment (jdalphon) [inline app/assets/stylesheets/projects.css.scss]: I didn't even know we were doing that.

**Pull request merged**